service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -55,9 +58,12 @@
     @Autowired
     private HearingVenueLookUpService hearingVenueLookUpService;
 
+    @MockBean
+    private UserDetailsService userDetailsService;
+
     private DateFormatterService dateFormatterService = new DateFormatterService();
     private HearingBookingService hearingBookingService = new HearingBookingService();
-    private DirectionHelperService directionHelperService = new DirectionHelperService();
+    private DirectionHelperService directionHelperService = new DirectionHelperService(userDetailsService);
     private HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration = new HmctsCourtLookupConfiguration(CONFIG);
```

Comment:
Not in your PR but stuff like this needs to be replaced with a test configuration so we don't have these awful setup methods

Reply:
👍